react/components/InstallmentsContext.ts

```diff
@@ -1,15 +1,8 @@
 import { createContext, useContext } from 'react'
-
-export interface Installments {
```

Comment:
This type as written as plural (installments) but it refer to a singular installment.